frontend/packages/dev-console/src/components/guided-tour/GuidedTourText.tsx

```diff
@@ -53,11 +51,12 @@ export const searchTourText = (
 );
 
 const FinishTourText: React.FC = () => {
-  const consoleLinks = useSelector<K8sResourceCommon[]>((state: RootState) =>
-    state.UI.get('consoleLinks'),
-  );
+  const [consoleLinks] = useK8sWatchResource<K8sResourceKind[]>({
+    isList: true,
+    kind: referenceForModel(ConsoleLinkModel),
+  });
   const openshiftBlogLink = consoleLinks.filter(
-    (link: K8sResourceCommon) => link.metadata.name === 'openshift-blog',
+    (link: K8sResourceKind) => link.metadata.name === 'openshift-blog',
```

Comment:
I think we probably want to just hard-code links like the OpenShift blog in console instead of using a `ConsoleLink` manifest in the operator repo. It's unnecessarily complicating the frontend logic, and we won't be able to translate the label.

cc @jhadvig

Reply:
I assume this is because there already is a `ConsoleLink` for the blog that's being used in the help menu?

Reply:
> I assume this is because there already is a `ConsoleLink` for the blog that's being used in the help menu?

That's right